lib/resources/r4/immunization_create.yaml

```diff
@@ -0,0 +1,301 @@
+---
+name: Immunization
+field_name_base_url: http://hl7.org/fhir/R4/immunization-definitions.html#Immunization
+fields:
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  url: http://hl7.org/fhir/r4/json.html#resources
+  description: The type of the FHIR resource.
+  action: create
+  example: |
+    {
+      "resourceType": "Immunization"
+    }
+  note: resourceType must be Immunization
+
+- name: status
+  required: 'Yes'
+  cardinality: 1..1
+  type: code
+  url: https://www.hl7.org/fhir/r4/immunization-definitions.html#Immunization.status
+  description: Indicates the current status of the immunization event.
+  action: create
+  example: |
+    {
+      "status": "completed"
+    }
+
+- name: statusReason
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  url: https://www.hl7.org/fhir/r4/immunization-definitions.html#Immunization.statusReason
+  description: Indicates the reason the immunization event was not performed.
+  action: create
+  example: |
+    {
+      "statusReason": {
+        "coding": [
+          {
+            "system": "https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/30440",
+            "code": "679607",
+            "display": "Patient Refuses",
+            "userSelected": true
+          }
+        ],
+        "text": "Patient Refuses"
+      }
+    }
+  note: This is generally only used for the status of "not-done".
+
+- name: vaccineCode
+  required: 'Yes'
+  cardinality: 1..1
+  type: CodeableConcept
+  url: https://www.hl7.org/fhir/r4/immunization-definitions.html#Immunization.vaccineCode
+  description: Vaccine that was administered or was to be administered.
+  action: create
+  example: |
+    {
+      "vaccineCode": {
+        "coding": [
+          {
+            "system": "https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/72",
+            "code": "2797921",
+            "display": "hepatitis A-hepatitis B vaccine",
+            "userSelected": false
+          }
+        ],
+        "text": "hepatitis A-hepatitis B vaccine"
+      }
+    }
+
+- name: patient
+  required: 'Yes'
+  cardinality: 1..1
+  type: Reference
+  url: https://www.hl7.org/fhir/r4/immunization-definitions.html#Immunization.patient
+  description: The patient who either received or did not receive the immunization.
+  action: create
+  example: |
+    {
+      "patient": {
+        "reference": "Patient/123820084",
+        "display": "Immun, Test1"
+      }
+    }
+
+- name: encounter
+  required: 'Yes'
+  cardinality: 0..1
+  type: Reference
+  url: https://www.hl7.org/fhir/r4/immunization-definitions.html#Immunization.encounter
+  description: The visit or admission or other contact between patient and health care provider the immunization was performed as part of.
+  action: create
+  example: |
+    {
+      "encounter": {
+          "reference": "Encounter/36209678"
```

Comment:
remove two extra spaces here and on L101

Reply:
https://github.com/cerner/fhir.cerner.com/pull/276/commits/6a039e09a31889bd9c0355262e750738c5a228a2#diff-6093e328274a82697fb90c9a404eda61L100